src/test/kotlin/com/devfloor/untitled/comment/application/CommentServiceTest.kt

```diff
@@ -0,0 +1,61 @@
+package com.devfloor.untitled.comment.application
+
+import com.devfloor.untitled.article.domain.Article
+import com.devfloor.untitled.article.domain.ArticleRepository
+import com.devfloor.untitled.comment.application.request.CommentRequest
+import com.devfloor.untitled.user.domain.User
+import com.devfloor.untitled.user.domain.UserRepository
+import com.devfloor.untitled.user.domain.UserType
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.assertAll
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.boot.test.context.SpringBootTest
+
+@SpringBootTest
+internal class CommentServiceTest {
+    @Autowired
+    lateinit var userRepository: UserRepository
+
+    @Autowired
+    lateinit var articleRepository: ArticleRepository
+
+    @Autowired
+    lateinit var commentService: CommentService
+
+    @Test
+    internal fun `create - 댓글 생성 테스트`() {
+        // given
+        val user = User(
+            nickname = "test",
+            type = UserType.STUDENT,
+            image = "image",
+            classOf = "B400000",
+            github = "github",
+            blog = "blog",
+            description = "description"
```

Comment:
여기 trailing comma 빠져있어!
```suggestion
            description = "description",
```

아 여기 말고도 테스트코드 아랫쪽 부분도 trailing comma 빠져있는 부분 채워넣으면 될거같아 !